components/In&OutputFields.js

```diff
@@ -0,0 +1,25 @@
+import styled from "styled-components";
+
+export default function Valuefields() {
+  return (
+    <Buttonsection>
+      <button>+50€</button>
+      <br></br>
```

Comment:
Don't use `<br>` to create space between your buttons. Use styling with styled components instead.